utils/src/main/java/ai/elimu/common/utils/di/TextToSpeechWrapper.kt

```diff
@@ -35,11 +35,11 @@ class TextToSpeechWrapper(context: Context, language: String, contentProviderId:
             Log.d(TAG, "init TextToSpeech DONE. status: $status")
             if (status == TextToSpeech.SUCCESS) {
                 tts.setSpeechRate(0.5f)
-                if (language.isNotEmpty()) {
-                    tts.setLanguage(language.toLanguage().toLocale())
-                } else {
-                    tts.setLanguage(contentProviderLanguage.toLanguage().toLocale())
+
+                val sttLanguage = language.ifEmpty {
```

Comment:
@tuancoltech The `language` variable here should probably be removed as a parameter? (Since the apps using this utils library don't know which language to use.)

```
class TextToSpeechWrapper(context: Context, language: String, contentProviderId: String)
```
-->
```
class TextToSpeechWrapper(context: Context, contentProviderId: String)
```

Reply:
@jo-elimu The `language` parameter here is for apps to be able to pass a custom TTS language to the TextToSpeech engine, disregard the one selected in ContentProvidre.
It's for the future.